quickstart/daml/licensing/daml/Licensing/AppInstall.daml

```diff
@@ -32,7 +32,6 @@ template AppInstall with
         params : LicenseParams -- ^ Parameters of the license to be created.
```

Comment:
@peterkvokacka-da : just saw that you still have the `dso` party in here. I don't think that's required anymore, as the payment instrument is determined by the provider on renewal.

Same holds likely for the `License` contract.